### ISSUE TITLE
ReadTheDocs builds are failing with missing "build.os" key.

### ISSUE THREAD

**Issue by bdice** [glotzerlab/freud#1155]
### Description

Failed build log: https://readthedocs.org/projects/freud/builds/21954536/

```
The configuration key "build.os" is required to build your documentation. Read more at https://docs.readthedocs.io/en/stable/config-file/v2.html#build-os
```

Blog post about this new requirement: https://blog.readthedocs.com/use-build-os-config/

### Steps to Reproduce

```python
n/a
```

### Error Output

```shell
n/a
```

### freud Version

n/a

### System Platform

None

### Installation method

None

### Developer

None

**Comment by bdice** [comment]
(The requirement to use an issue template is kind of frustrating, since this is a bug that doesn't fit the usual format. I would have liked to file a blank issue with this information)

**Comment by tommy-waltmann** [comment]
Addressed in #1158
